src/components/Header/Header.tsx

```diff
@@ -1,11 +1,69 @@
-import React from 'react';
+import React, { FC } from 'react';
+import './header.scss';
+import NiceGadgetsLogo from '../../img/Nice Gadgets.png';
+import Favourite from '../../img/icons/favourite.svg';
+import Basket from '../../img/icons/shopping_bag.svg';
 import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
```

Comment:
Let's make a rearrangement: important react imports first, then classnames, images etc. Just to keep some order :)